utils/api/definitions/fields.ts

```diff
@@ -91,7 +91,7 @@ export const fieldDefinitions: FieldDefinitions = {
   [FieldKey.EVER_LIVED_SOCIAL_COUNTRY]: {
     key: FieldKey.EVER_LIVED_SOCIAL_COUNTRY,
     label:
-      'Have you ever lived in a country with an established social security agreement?',
+      'Have you lived in a country with an established <a href="https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/payroll/payroll-deductions-contributions/canada-pension-plan-cpp/foreign-employees-employers/canada-s-social-agreements-other-countries.html" target="_blank">social security agreement</a> before?',
     category: FieldCategory.LEGAL_STATUS,
```

Comment:
This kind of snuck in here but it is pretty innocuous so I think it's fine to leave in this PR. Not really part of error validation though :P